### ISSUE TITLE
Horizontal status bars

### ISSUE THREAD

**Issue by ThimbleFire** [runelite/runelite#8516]
I'd like to suggest horizontal status bars for a more traditional RPG appearance.

![default](https://i.imgur.com/Nn2znR9.png)

![default](https://i.imgur.com/S63cEMd.png)

I realise however that it defeats the point of them being docked next to the inventory, where view on them isn't obscured. With that being said I still think it'd be nice to have this option.

**Comment by Nightfirecat** [comment]
If an option is added to let them display horizontally, I think it would also be reasonable for the overlay to be draggable when displaying that way. With that said, I think an argument can be made for making the stacked-tabs view of the status bars draggable as well.

**Comment by Beagon** [comment]
@Nightfirecat Is there any easy example of how to make a overlay draggable? If I can find and example I'd be glad to take this one on me.

**Comment by Malfuryent** [comment]
i would love to look into this too @Beagon currently i have pr #7730 ready for the statusbars which nobody currently looked at anymore. i'd like to push that one first. 

i could add a mode to make them horizontal as a seperate feature but that would make them be on static spots like the way i implemented them now so not draggable.

**Comment by Beagon** [comment]
> i would love to look into this too @Beagon currently i have pr #7730 ready for the statusbars which nobody currently looked at anymore. i'd like to push that one first.
> 
> i could add a mode to make them horizontal as a seperate feature but that would make them be on static spots like the way i implemented them now so not draggable.

Very nice, I'll let you to it then :)

**Comment by Malfuryent** [comment]
Yeah sure that's fine if you want to look into it. but keep in mind that 7730 introduces alot of changes to the plugin, i dont know how much this could affect your efforts, goodluck :)

**Comment by Beagon** [comment]
I'll just wait 'till that one is merged then. Also would still love to find a good example of how to make the widget movable. I've been trying to look in the code of other plugins but to no avail...

**Comment by Nightfirecat** [comment]
Making an overlay moveable is just a matter of setting an overlay's position. That said, I don't think we have any examples of overlays which can be moved sometimes but not other times.

**Comment by raiyni** [comment]
I don't there would be a good position to put horizontal health bars considering their size.

**Comment by Beagon** [comment]
> 
> 
> I don't there would be a good position to put horizontal health bars considering their size.

All depends on your setup to be honest, it wouldn't look good in non-resizable indeed but for my setup it would work perfectly.



> 
> 
> Making an overlay moveable is just a matter of setting an overlay's position. That said, I don't think we have any examples of overlays which can be moved sometimes but not other times.

Alright, I'm going to try and tinker around with it a little to see if I can get something to work. Thought it might be as easy as making it an infobox or something when in the right mode hehe
